app/controllers/users/comments_controller.rb

```diff
@@ -1,27 +1,26 @@
 module Users
-  class CommentsController < ApplicationController
-    respond_to :json
+  class CommentsController < Users::BaseController
```

Comment:
You can simply write `class CommentsController < BaseController`, because you are already in `Users` module